### PR TITLE
Bats test on server passed.

### PR DESCRIPTION
**Pull request by rubenchazarra**
Test on server passed. This PR includes some of the most used functions for scran 1.12. 
Some functions from scran 1.14 are also included as scripts, but outcomented in bats test. Will be updated soon.

### CODE REVIEW

File: dropletutils-read-10x-counts.R [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env Rscript
```
Comment (pinin4fjords) [inline dropletutils-read-10x-counts.R]: Remember you don't need to duplicate this script. Use it from the dropletutils-scripts package :-)

File: get_experiment_data.R [hunk 2]
```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env Rscript
```
Comment (pinin4fjords) [inline get_experiment_data.R]: This is Andrey's script, right? Again, use it from the Conda package.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,110 @@
...
-Command line interface for scran package. Low-level analyses of single-cell RNA-seq data.
+# Wrapper scripts for components of the scran package
+
+Scran implements functions for low-level analyses of single-cell RNA-seq data. Methods are provided for normalization of cell-specific biases, assignment of cell cycle phase, detection of highly variable and significantly correlated genes, identification of marker genes, and other common tasks in routine single-cell analysis workflows.
+
+A vignette with the usage of scran can be found [here](https://bioconductor.org/packages/release/bioc/vignettes/scran/inst/doc/scran.html) 
+
+**Note**: this scripts are for Scran version 1.12. 
+
+## Install
```
Comment (a-solovyev12) [inline README.md:14]: @rubenchazarra,  you can add some notes on how to install the package even though it's not on Bioconda yet - something like `conda install scran-cli`. Also good to mention that it is better to install into a clean environment

File: scran-compute-sum-factors.R [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    default = NA,
+    type = 'character',
+    help = 'Path to the input SCE object in rds format'
+  ),
+    make_option(
+    c("-a", "--assay-type"),
+    action = "store",
+    default = "counts",
+    type = 'character',
+    help = 'Specify which assay values to use. Default: "logocunts'
```
Comment (a-solovyev12) [inline scran-compute-sum-factors.R]: @rubenchazarra, 'logocunts' is a nice one, but you'll have to replace it, I'm afraid ;)

File: scran-cli-filter-sce.R [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+}    
+
+#check number of GENES to sample is not larger that available
+if(ncol(sce) > opt$n_cells){
+    sce <- sce[, sample(colnames(sce), opt$n_cells)]
+}
+
+#add some genes as spike-ins, so that scran_computeSpikeFactors.R can run without error
+suppressPackageStartupMessages(require(SingleCellExperiment))
+sce <- `isSpike<-`(sce, type = "ERCC", value = c(sample(rownames(sce), 25)))
```
Comment (a-solovyev12) [inline scran-cli-filter-sce.R]: could you parameterise this line so that spike-ins are only added when specified by user?

**Review by a-solovyev12 — APPROVED** [review]

**Comment by a-solovyev12** [comment]
looks good to me - thanks @rubenchazarra

**Review by pinin4fjords — APPROVED** [review]

**Pull request merged**